lib/SparkPost/Transmission.php

```diff
@@ -89,6 +89,11 @@ class Transmission extends APIResource {
    * @return array API repsonse represented as key-value pairs
    */
   public function send( $transmissionConfig ) {
+    if(isset($transmissionConfig["startTime"]) && $transmissionConfig["startTime"] instanceof \DateTime)
+      {
+        $transmissionConfig["startTime"] = $transmissionConfig["startTime"]->format(\DateTime::ATOM);
```

Comment:
Same here. Should we support this as a base functionality for all DateTimes?  Moving this to the APIResource class might be better.